router/router.go

```diff
@@ -1148,6 +1145,19 @@ func (rt *HandleT) findWorker(job *jobsdb.JobT, throttledAtTime time.Time) (toSe
 		return nil
 	}
 
+	err := json.Unmarshal(job.Parameters, &parameters)
+
+	if err == nil && rt.canThrottle(parameters.DestinationID, userID, throttledAtTime) {
```

Comment:
1. I prefer if error checking is seperated from logic, and ideally imditly after method invocation. Requires less cognitive effort to read it.

2.  not introduce in this PR but, `shouldThrottle` seems more suitable than `canThrottle`.
```suggestion
	err := json.Unmarshal(job.Parameters, &parameters)
    if err != nil {
        // error log
        return nil
    }
	if rt.shouldThrottle(parameters.DestinationID, userID, throttledAtTime) {
```